### PR TITLE
Add CR claims DPOS node transactions

### PR DESCRIPTION
**Pull request by peterwillcn**
Signed-off-by: Xiaobo Tian <peterwillcn@gmail.com>

### CODE REVIEW

File: core/types/payload/crdposmanagement.go [hunk 1]
```diff
@@ -0,0 +1,167 @@
+package payload
+
+import (
+	"bytes"
+	"errors"
+	"github.com/elastos/Elastos.ELA/common"
+	"github.com/elastos/Elastos.ELA/crypto"
+	"io"
```
Comment (RainFallsSilent) [inline core/types/payload/crdposmanagement.go:8]: need to separate package

File: core/types/payload/crdposmanagement.go [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+	}
+}
+
+type CRDPOSManagement struct {
+	ManagementType        DPOSManagementType
+	CRManagementPublicKey []byte
+	CRCommitteeDID        common.Uint168
+	Signature             []byte
+	hash                  *common.Uint256
+}
```
Comment (RainFallsSilent) [inline core/types/payload/crdposmanagement.go:33]: no need hash

File: core/types/payload/crdposmanagement.go [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+type DPOSManagementType uint16
+
+func (t DPOSManagementType) Name() string {
+	switch t {
+	case CRManagement:
+		return "CRManagement"
+	default:
+		return "Unknown"
+	}
+}
```
Comment (RainFallsSilent) [inline core/types/payload/crdposmanagement.go:25]: no need to use switch

File: core/types/payload/crdposmanagement.go [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+func (p *CRDPOSManagement) DeserializeUnSigned(r io.Reader, version byte) error {
+	switch p.ManagementType {
+	case CRManagement:
+		return p.DeserializeUnSignedCRManagement(r, version)
+	default:
+		return p.DeserializeUnSignedCRManagement(r, version)
+	}
+}
```
Comment (RainFallsSilent) [inline core/types/payload/crdposmanagement.go:145]: no need to use switch

File: core/types/payload/crdposmanagement.go [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+		return errors.New("[CRDPOSManagement], ManagementType deserialize failed")
+	}
+
+	switch p.ManagementType {
+	case CRManagement:
+		return p.DeserializeCRManagement(r, version)
+	default:
+		return p.DeserializeCRManagement(r, version)
+	}
+}
```
Comment (RainFallsSilent) [inline core/types/payload/crdposmanagement.go:112]: no need to use switch

File: core/types/payload/crdposmanagement.go [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+	return buf.Bytes()
+}
+
+func (p *CRDPOSManagement) SerializeUnsigned(w io.Writer, version byte) error {
+	switch p.ManagementType {
+	case CRManagement:
+		return p.SerializeUnsignedCRManagement(w, version)
+	default:
+		return p.SerializeUnsignedCRManagement(w, version)
+	}
```
Comment (RainFallsSilent) [inline core/types/payload/crdposmanagement.go:60]: no need to use switch

Comment (peterwillcn) [reply core/types/payload/crdposmanagement.go:60]: Reserve for future expansion

File: core/types/payload/crdposmanagement.go (hunk 5 above)
Comment (peterwillcn) [reply core/types/payload/crdposmanagement.go:112]: Reserve for future expansion

File: core/types/payload/crdposmanagement.go (hunk 4 above)
Comment (peterwillcn) [reply core/types/payload/crdposmanagement.go:145]: Reserve for future expansion

File: core/types/payload/crdposmanagement.go (hunk 3 above)
Comment (peterwillcn) [reply core/types/payload/crdposmanagement.go:25]: Reserve for future expansion

**Pull request merged**